crates/bitwarden-vault/src/cipher/attachment.rs

```diff
@@ -4,13 +4,15 @@ use bitwarden_crypto::{
 };
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
+use tsify_next::Tsify;
```

Comment:
We also want the Tsify import to be behind a feature so it's not imported for other targets
```suggestion
#[cfg(feature = "wasm")]
use tsify_next::Tsify;
```

This also applies to the other files